src/frontend/context/InnloggetContext.ts

```diff
@@ -33,7 +32,6 @@ const [InnloggetProvider, useInnloggetContext] = createUseContext(() => {
         return axiosRequest({
             url: `${soknadApiProxyUrl}/innlogget/barnetrygd`,
             method: 'GET',
-            params: { søknadstype: hentSøknadstype() },
```

Comment:
Her må vi også inn i APIet og endre tellerne for antall innloggede i ordinær/utvidet. Fra vi skrudde på toggle er ikke søknadstype bestemt ennå når dette kallet gjøres.